changelogs/unreleased/ghs-116-possible-segfault-in-misc.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
+
+* Fixed possible segmentation fault due to unvalidated return value of
+  region_alloc() (ghs-116).
```

Comment:
Let's drop the changelog? The bug shouldn't normally happen. And replacing segfault with panic doesn't look like a fix, actually. So this would only clutter the changelog.